### ISSUE TITLE
Update "Dodge the Creeps 2D" documentation for godot 4.0 release

### ISSUE THREAD

**Issue by Lucksinia** [godotengine/godot-docs#6935]
**Your Godot version:**
v4.0.stable.official [92bee43ad]
**Issue description:**
So.... "Your first 2d game" is outdated to the point of error. What do i mean by that?
1. There is ancient Input vector config as mentioned in #6687
2. There is errors in "Creating the enemy" page. I am talking about Enemy script in particular:
```
func _ready():
    $AnimatedSprite2D.play()
    var mob_types = $AnimatedSprite2D.get_sprite_frames().get_animation_names()
    $AnimatedSprite2D.animation = mob_types[randi() % mob_types.size()]
```
Where we can see some deprecated function naming (**get_sprite_frames** became  **sprite_frames** in this particular case).
And I believe there is more outdated tricks and code solutions, but I cannot judge that matter because i picked up Godot Engine on the 1 of March 2023.

I propose to update both "Your first..." Documentation pages to contain latest "good practice" implementations of  their code base.

**URL to the documentation page (if already existing):**
*Creating the enemy:*
https://docs.godotengine.org/en/stable/getting_started/first_2d_game/04.creating_the_enemy.html#enemy-script
*Coding the player:*
https://docs.godotengine.org/en/stable/getting_started/first_2d_game/03.coding_the_player.html

**Comment by Piralein** [comment]
The specific problem with `get_sprite_frames()` was fixed by [#6811](https://github.com/godotengine/godot-docs/pull/6811) in master and will be merged/cherry picked at some point to stable.

**Comment by Lucksinia** [comment]
Well, It didn't worked for me, so...
